x-pack/plugins/apm/public/components/shared/suggestions_select/index.tsx

```diff
@@ -91,17 +98,19 @@ export function SuggestionsSelect({
   return (
     <EuiComboBox
       async={true}
-      compressed={true}
       customOptionText={customOptionText}
+      isClearable={isClearable}
       isLoading={status === FETCH_STATUS.LOADING}
       onChange={handleChange}
-      onCreateOption={handleCreateOption}
       onSearchChange={debounce(setSearchValue, 500)}
```

Comment:
I think we should be using `throttle` instead of debounce. We want to provide continuous results back to the user - not necessarily wait until they are finished typing.

Reply:
I updated and yes the ui experience feels much better to me
https://github.com/elastic/kibana/pull/122648/commits/e7e068a12e3596b2547a8fe8855153a9302774b9